pkg/alerts/nrql_conditions.go

```diff
@@ -415,6 +427,26 @@ func (a *Alerts) UpdateNrqlConditionStaticMutation(
 	return &resp.AlertsNrqlConditionStaticUpdate, nil
 }
 
+// CreateNrqlConditionOutlierMutation creates an outlier type NRQL alert condition via New Relic's NerdGraph API.
+func (a *Alerts) CreateNrqlConditionOutlierMutation(
+	accountID int,
+	policyID int, // DEV NOTE: policyID is the gql scalar type ID, so per team agreement, need to update to strings for this I think
```

Comment:
Reminder: Inquire about converting scalar `ID` properties to strings since that's how they're returned in the JSON responses.